src/components/Button/Button.jsx

```diff
@@ -1,6 +1,13 @@
 import './Button.css'
+import PropTypes from 'prop-types'
 
-const Button = ({onClick, label}) =>
-    <button className="button" onClick={onClick}>{label}</button>
+const Button = ({onClick, label, className}) =>
+    <button className={`button ${className}`} onClick={onClick}>{label}</button>
```

Comment:
for managing classnames better to use 'classnames' library

https://github.com/JedWatson/classnames